index.html

```diff
@@ -4,714 +4,895 @@
-                Post a vacancy
-              </a>
-            </div>
-          </nav>
-          <!-- Hero section wrapper -->
-          <div class="hero-section__wrapper">
-            <!-- Left wrapper -->
-            <div class="left-wrapper">
-              <h1 class="left-wrapper__title title">
-                Find & Apply For Your Dream Job
-              </h1>
-              <p class="left-wrapper__subtitle">
-                The website is chockablock with useful features, allowing you to
-                efficiently search and apply for roles across all sectors,
-                experience levels and locations.
-              </p>
+      <!-- Hamburger -->
+      <button class="hamburger">
+        <span class="hamburger__item"></span>
+        <span class="hamburger__item"></span>
+        <span class="hamburger__item"></span>
+      </button>
 
-              <!-- Search container -->
-              <div class="search-container">
-                <!-- Search form -->
-                <form class="search-container__search-form" method="get">
-                  <img
-                    class="search-container__search-icon"
-                    src="./src/icons/search-icon.svg"
-                    alt="search"
-                  />
-                  <input
-                    class="search-input"
-                    type="text"
-                    placeholder="Job title or Keyword"
-                    name="text"
-                    value=""
-                  />
-                </form>
+      <div class="container header-wrapper">
+        <a class="logo" href="#"> J<span class="decor_orange">O</span>BPLY </a>
 
-                <!-- Location form -->
-                <img src="./src/icons/line.svg" alt="icon" />
-                <form class="search-container__location-form" method="get">
-                  <img
-                    class="search-container__location-icon"
-                    src="./src/icons/location-icon.svg"
-                    alt="location"
-                  />
-                  <input
-                    class="location-input"
-                    type="text"
-                    placeholder="Location"
-                    name="text"
-                    value=""
-                  />
-                </form>
+        <!-- Navigation menu -->
+        <nav class="nav-menu">
+          <ul class="nav-menu__list">
+            <li class="list-item"><a href="#">Home</a></li>
+            <li class="list-item">
+              <a href="#">Categories</a>
+            </li>
+            <li class="list-item"><a href="#">Service</a></li>
+            <li class="list-item"><a href="#">Jobs</a></li>
+            <button class="button nav__button" href="#">Post a vacancy</button>
+          </ul>
+        </nav>
+        <nav class="menu nav-menu">
+          <!-- Navigation list items -->
+          <ul class="menu__list">
+            <li class="list-item">
```

Comment:
```suggestion
            <li class="menu__list-item">
```
<li> is always a part of a list and can't be used outside, so the class should show that it's an element